tools/travis-build-tests.sh

```diff
@@ -12,5 +12,22 @@ export TRAVIS_COMMIT_MESSAGE=no_message
 
 if [ "$BUILD_TESTS" != "false" ]; then
     cd ${BASE}/big_tests && make prepare
+
+    # Copy data files, so tests, compiled by rebar still
+    # can find the data.
+    #
+    # Common Tests use filename from beam file to figure out data_dir.
+    # data_dir for rebar compiled files would be somewhere in _build directory,
+    # not where we would expect it to be.
+    #
+    # Used with SKIP_AUTO_COMPILE=true
+    mkdir -p _build/default/lib/ejabberd_tests/ebin/
+    for i in tests/*_data; do
+        echo "$i"
+        base=$(basename "$i")
+        dest="_build/default/lib/ejabberd_tests/ebin/$base"
+        rm -f "$dest"
+        ln -s "../../../../../$i" "$dest"
```

Comment:
I know, it sounds stupid, but we call `tools/travis-build-tests.sh` from circle-ci AND test-runner.

I know, it also stupid to have this .......... path, but it would not change soon for sure XD (unless rebar4)